### DISCUSSION TITLE
velero + minio: Error getting volume snapshotter for volume snapshot location

### DISCUSSION THREAD

**Discussion by kiaz-github** [vmware-tanzu/velero#4739] [Community support Q&A]
The backup is completed (all items are backed up) but the status is "partially failed" because of this error:

time="2022-03-14T15:39:03Z" level=error msg="Error getting volume snapshotter for volume snapshot location" backup=velero/testbackup07 error="rpc error: code = Unknown desc = missing region in aws configuration" error.file="/go/src/velero-plugin-for-aws/velero-plugin-for-aws/volume_snapshotter.go:76" error.function="main.(*VolumeSnapshotter).Init" logSource="pkg/backup/item_backupper.go:444" name=pvc-7078f34b-a748-4543-bbeb-70eb4240911f namespace= persistentVolume=pvc-7078f34b-a748-4543-bbeb-70eb4240911f resource=persistentvolumes volumeSnapshotLocation=default

I installed Velero server v1.8.0 using vmware-tanzu/velero helm chart.
I installed Minio v2022-03-11T23-57-45Z (latest release).

This is the velero-values.yaml file:

`initContainers:
  - name: velero-plugin-for-aws
    image: velero/velero-plugin-for-aws:v1.3.1
    imagePullPolicy: Always
    volumeMounts:
    - mountPath: /target
      name: plugins
credentials:
  useSecret: true
  secretContents:
    cloud: |
      [default]
      aws_access_key_id=xxx
      aws_secret_access_key=xxx
configuration:
  provider: aws
  backupStorageLocation:
    bucket: backup-bucket
    config:
      region: minio
      s3ForcePathStyle: true
      publicUrl: xxxxxxx
      s3Url: http://minio.velero.svc:9000
snapshotsEnabled: false
`

This is the backup description:

Name:         testbackup07
Namespace:    velero
Labels:       velero.io/storage-location=default
Annotations:  velero.io/source-cluster-k8s-gitversion=v1.23.1
              velero.io/source-cluster-k8s-major-version=1
              velero.io/source-cluster-k8s-minor-version=23

Phase:  PartiallyFailed (run `velero backup logs testbackup07` for more information)

Errors:    1
Warnings:  0

Namespaces:
  Included:  xxx
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Label selector:  <none>

Storage Location:  default

Velero-Native Snapshot PVs:  auto

TTL:  720h0m0s

Hooks:  <none>

Backup Format Version:  1.1.0

Started:    2022-03-14 16:39:02 +0100 CET
Completed:  2022-03-14 16:39:03 +0100 CET

Expiration:  2022-04-13 17:39:02 +0200 CEST

Total items to be backed up:  32
Items backed up:              32

**Comment by inzanez** [comment]
Did you ever manage to solve this issue? ;-)
